src/Key.php

```diff
@@ -10,13 +10,92 @@ class Key
     /** @var string */
     public $secret;
 
+    /** @var string */
+    public $certificate;
+
+    /** @var string */
+    public $privateKey;
+
+    /** @var string */
+    public $type;
+
     /**
      * @param string $id
      * @param string $secret
      */
-    public function __construct($id, $secret)
+    public function __construct($id, $item)
     {
         $this->id = $id;
-        $this->secret = $secret;
+        $certificate = Key::getX509Certificate($item);
+        $privateKey = Key::getRSAPrivateKey($item);
```

Comment:
`static::getRSAPrivateKey()`?

Reply:
It's independent of any instance state, so felt it could be static. Fixed in c2b116052048e805b6fc339cf95e80693de2eee9